stable/redis/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+Redis can be accessed via port 3306 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To get your password run:
+
+    REDIS_PASSWORD=$(printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.redis-password[*]}"`))
```

Comment:
Should we put this in a configmap? Secrets are not really secret, anyone that knows the name of a secret can access said secret. Not sure if configmap works better